core/android_make.go

```diff
@@ -153,7 +153,7 @@ func specifyCompilerStandard(varname string, flags []string) string {
 // This function generates the Android make fragment to build static
 // libraries, shared libraries and executables. It's evolved over time
 // and needs to be refactored to use interfaces better.
-func androidLibraryBuildAction(sb *strings.Builder, mod blueprint.Module, ctx blueprint.ModuleContext) {
+func androidLibraryBuildAction(tcs toolchainSet, sb *strings.Builder, mod blueprint.Module, ctx blueprint.ModuleContext) {
```

Comment:
Please move the toolchainSet to the end of the argument list.

The other arguments are the primary things that we are operating on, and the toolchain is secondary to what the function is doing.

Reply:
ok.